### PR TITLE
Added changes to upload performance kpi data

### PR DESCRIPTION
**Pull request by tisutisu**
Adding support to upload performance kpi data to the open search as part of story: https://issues.redhat.com/browse/APPSVC-1261

### CODE REVIEW

**Comment by pmacik** [comment]
/hold
Depends on https://github.com/redhat-developer/service-binding-operator/pull/1331

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml [hunk 1]
```diff
@@ -83,9 +83,11 @@ tests:
         export TEST_PERFORMANCE_USERS_PER_SCENARIO=400
         export REGISTRY_NAMESPACE=ocpappsvc-qe
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
+        source /var/run/secrets/sbo-on-sb-perf/opensearch-aws-credentials-env.sh
         make -k test-performance test-performance-collect-kpi || true
         make test-performance-artifacts
-        make test-performance-thresholds
+        make test-performance-thresholds || true
+        make test-performance-upload-kpi
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:90]:
```suggestion
        make test-performance-upload-kpi || true
        make test-performance-thresholds
```
This step is supposed to fail the job if the thresholds are exceeded and so it should be the last step in the job and without the `|| true`.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml [hunk 2]
```diff
@@ -121,9 +121,11 @@ tests:
         export TEST_PERFORMANCE_USERS_PER_SCENARIO=400
         export REGISTRY_NAMESPACE=ocpappsvc-qe
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
+        source /var/run/secrets/sbo-on-sb-perf/opensearch-aws-credentials-env.sh
         make -k test-performance test-performance-collect-kpi || true
         make test-performance-artifacts
-        make test-performance-thresholds
+        make test-performance-thresholds || true
+        make test-performance-upload-kpi
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml:128]:
```suggestion
        make test-performance-upload-kpi || true
        make test-performance-thresholds
```
This step is supposed to fail the job if the thresholds are exceeded and so it should be the last step in the job and without the `|| true`.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.12.yaml [hunk 3]
```diff
@@ -99,9 +99,11 @@ tests:
         export TEST_PERFORMANCE_USERS_PER_SCENARIO=400
         export REGISTRY_NAMESPACE=ocpappsvc-qe
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
+        source /var/run/secrets/sbo-on-sb-perf/opensearch-aws-credentials-env.sh
         make -k test-performance test-performance-collect-kpi || true
         make test-performance-artifacts
-        make test-performance-thresholds
+        make test-performance-thresholds || true
+        make test-performance-upload-kpi
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.12.yaml:106]:
```suggestion
        make test-performance-upload-kpi || true
        make test-performance-thresholds
```
This step is supposed to fail the job if the thresholds are exceeded and so it should be the last step in the job and without the `|| true`.

**Review by pmacik — CHANGES_REQUESTED** [review]

**Comment by pmacik** [comment]
/unhold
https://github.com/redhat-developer/service-binding-operator/pull/1331 is merged

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml (hunk 1 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml]: changed the sequence now in the latest commit

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml (hunk 2 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml]: changed the sequence now in the latest commit

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.12.yaml (hunk 3 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.12.yaml]: changed the sequence now in the latest commit

**Comment by pmacik** [comment]
Depends on https://github.com/redhat-developer/service-binding-operator/pull/1335

**Pull request merged**